### PR TITLE
Add node.js support

### PR DESCRIPTION
**Pull request by Rexogamer**
#21 is semi-closed by this.

Ad support for node.js.

### CODE REVIEW

File: project_types.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# process for react typescript projects
+def ReactTS():
+    subprocess.run(f"npx create-react-app {script.projectName} --typescript", shell=True)
+    os.chdir(script.projectName)
+
+    
+# process for nodejs projects
+def NodeJS():
+    subprocess.run(f"npm init {script.projectName}", shell=True)
+    od.chdir(script.projectName)
```
Comment (jarodburchill) [inline project_types.py]: od = overdose? lol

**Review by jarodburchill — COMMENTED** [review]
I'll merge the node stuff into my own branch then you can pull from my branch and keep going from there if you'd like?

File: project_types.py (hunk 1 above)
Comment (Rexogamer) [reply project_types.py]: whoops

**Comment by Rexogamer** [comment]
sure, but idk how to proceed with python

**Comment by Rexogamer** [comment]
Saw your commit on your branch

**Pull request closed without merging**